x-pack/plugins/ingest_manager/server/services/epm/agent/agent.ts

```diff
@@ -87,6 +87,14 @@ function buildTemplateVariables(variables: PackageConfigConfigRecord, streamTemp
   return { vars, yamlValues };
 }
 
+function containsHelper(item: string, list: string[], options: any) {
+  if (list instanceof Array && list.indexOf(item) > -1) {
+    return options.fn(this);
+  }
+  return '';
+}
+Handlebars.registerHelper('contains', containsHelper);
```

Comment:
We probably need to create a dedicated Handlebars environement `Handlebars.create()`

Reply:
ah, thanks for that, was trying to figure out how to do so rummaging through handlebars docs so we wouldn't have to register globally